contracts/interfaces/IERC4337Account.sol

```diff
@@ -4,6 +4,8 @@ pragma solidity ^0.8.24;
 import { PackedUserOperation } from "account-abstraction/contracts/interfaces/PackedUserOperation.sol";
 
 interface IERC4337Account {
+    event DepositAdded(address indexed account, address indexed depositor, uint256 amount);
```

Comment:
all three can be indexed

Reply:
that's really a bad practice to index amount.